common-rest/src/test/java/org/uniprot/api/common/repository/search/SolrRequestConverterTest.java

```diff
@@ -162,15 +162,24 @@ void requestingTermFieldsWithoutTermQueryCausesException() {
         }
 
         @Test
-        void tooLongQueryWithTermsRequestCausesException() {
+        void doNotAddTermParametersForPhrasalQueryTermRequest() {
```

Comment:
This isn't a phrase query, because the query would need to be encapsulated by quotes.